amy/workshops/lookups.py

```diff
@@ -79,7 +89,15 @@ def get_queryset(self):
 
 class TTTEventLookupView(OnlyForAdminsNoRedirectMixin, AutoResponseView):
     def get_queryset(self):
-        results = models.Event.objects.filter(tags__name="TTT")
+        results = models.Event.objects.ttt()
+
+        # trainee is provided through the TrainingProgress creation views
+        # for which learner tasks are relevant
+        if trainee := self.request.GET.get("trainee"):
+            learner_tasks = models.Task.objects.filter(
+                role__name="learner", person__id=trainee
+            )
+            results = results.filter(task__in=learner_tasks)
```

Comment:
Is this a similar case as above? Perhaps we should have a separate class for this case, too.

Reply:
This lookup is only used in the training views (TrainingProgressForm, BulkAddTrainingProgressForm, TraineeFilter), so I'm not sure it makes as much sense to separate it out.